src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentService.kt

```diff
@@ -54,7 +54,7 @@ class SentencingAdjustmentService(
   fun createSentenceAdjustment(bookingId: Long, sentenceSequence: Long, request: CreateSentenceAdjustmentRequest) =
     offenderBookingRepository.findByIdOrNull(bookingId)?.let { booking ->
       offenderSentenceRepository.findByIdOrNull(SentenceId(booking, sentenceSequence))?.let { sentence ->
-        sentence.adjustments.add(
+        val adjustmentId = offenderSentenceAdjustmentRepository.save(
```

Comment:
save adjustment directly rather than adding to list and saving parent, we can't guarantee that `adjustments.last()` below will return this new one or another created at exact same time